app/src/main/java/com/teamltt/carcare/activity/ReminderActivity.java

```diff
@@ -101,40 +135,118 @@ protected void updateUi(final Cursor info) {
         Button button;
 
         while (info.moveToNext()) {
```

Comment:
This `info` variable is of `Cursor` type which should not exist beyond the `DbHelper` class.